src/useFormState.js

```diff
@@ -71,6 +81,7 @@ export default function useFormState(initialState, options) {
 
     const inputProps = {
       name,
+      id: idGetter(name, ownValue),
```

Comment:
Due to #26, we need to call `toString` here

```suggestion
      id: idGetter(name, toString(ownValue)),
```